target_chains/aptos/sdk/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/pyth-aptos-js",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
nothing has changed code-wise, but i want to make sure this one and the one below get published